_pages/conduct.html

```diff
@@ -21,25 +22,19 @@ <h1>{{ page.heading }}</h1>
   <section class="section-pad">
     <div class="row column">
         <h2 class="demp">
-        DjangoCon US is dedicated to providing a fun harassment-free conference experience for everyone, regardless of gender, gender identity, sexual orientation, disability, physical appearance, body size, race, or religion.
+          We ask each attendant to keep the community in mind and do their best to foster a positive environment for everyone.
```

Comment:
All content here is a copy of https://2022.djangocon.eu/conduct/code_of_conduct/.